ci-operator/step-registry/openshift/e2e/ibmcloud/heterogeneous/cucushift/openshift-e2e-ibmcloud-heterogeneous-cucushift-workflow.yaml

```diff
@@ -0,0 +1,19 @@
+workflow:
+  as: openshift-e2e-ibmcloud-heterogeneous-cucushift
```

Comment:
```suggestion
  as: openshift-e2e-ibmcloud-heterogeneous-test-qe
```

Reply:
@liangxia is this the change you are suggesting...

Reply:
OCP QE have a readme in https://github.com/openshift/release/blob/master/ci-operator/config/openshift/openshift-tests-private/README.md#job-naming